src/components/VideoPlayer/VideoPlayer.component.jsx

```diff
@@ -11,8 +12,33 @@ import {
   StyledVideoLike,
 } from './styled';
 
-function VideoPlayer({ videoId, title, description }) {
+function VideoPlayer({ videoId, title, description, thumbnail }) {
   const videRole = 'youtube-video';
+  const { addToFavorites, favorites, deleteFavorites } = useAuth();
+  const checkAlreadyFavorite = (currentId) => {
+    let found = false;
+    favorites.map((item) => {
```

Comment:
There's a method you can use for arrays called `find` which looks for an item in the array. Check this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find